src/webapp/reports/autogenerated-forms/GridWithCatOptionCombos.tsx

```diff
@@ -0,0 +1,128 @@
+
+    return (
+        <DataTableSection section={grid} dataFormInfo={dataFormInfo}>
+            <DataTable className={classes.table} layout="fixed" width="initial">
+                <TableHead>
+                    <DataTableRow>
+                        <DataTableColumnHeader width="400px" colSpan="2"></DataTableColumnHeader>
+
+                        {grid.columns.map(column => (
+                            <DataTableColumnHeader key={column.name}>
+                                <span>{column.name}</span>
+                            </DataTableColumnHeader>
+                        ))}
+                    </DataTableRow>
+                </TableHead>
+
+                <TableBody>
+                    {grid.rows.map(row => {
+                        const groupName = row.groupName;
+                        const rows = row.rows;
+
+                        return rows.map((row, idx) => (
+                            <DataTableRow key={`${groupName}-${idx}`}>
+                                {groupName ? (
+                                    <>
+                                        {idx === 0 && (
+                                            <DataTableCell
+                                                className={classes.rowTitle}
+                                                rowSpan={rows.length.toString()}
+                                            >
+                                                <span>{groupName}</span>
+                                            </DataTableCell>
+                                        )}
+
+                                        <DataTableCell>
+                                            <span>{row.deName}</span>
+                                        </DataTableCell>
+                                    </>
+                                ) : (
+                                    <DataTableCell colSpan="2">
+                                        <span>{row.deName}</span>
+                                    </DataTableCell>
+                                )}
+
+                                {grid.columns.map(column => {
+                                    const dataElement = column.items.find(item =>
+                                        item.row.rows.map(row => row.deName).includes(row.deName)
+                                    )?.dataElement;
+
+                                    if (dataElement) {
+                                        return (
+                                            <DataTableCell key={[dataElement.id, column.name].join("-")}>
+                                                <DataElementItem
+                                                    dataElement={dataElement}
+                                                    dataFormInfo={dataFormInfo}
+                                                    noComment={
+                                                        dataElement.name !== "Source type for HWF - (Inputs & Outputs)"
```

Comment:
This validation was for other viewType. Since we don't want to show the comments let's just pass `true` or leave the prop without any value.

```tsx
  <DataElementItem 
      dataElement={dataElement}
      dataFormInfo={dataFormInfo}
      // same as noComment={true}
      noComment
```